src/components/InputEntry.tsx

```diff
@@ -1,16 +1,17 @@
 /* eslint-disable react/require-default-props */
-import type { ReactNode } from 'react';
+import type { PropsWithoutRef, ReactNode } from 'react';
 import { forwardRef } from 'react';
 import { Element } from 'react-scroll';
 
 import InputErrorMessage from 'components/InputErrorMessage';
 import { Heading, TextInput } from 'design-system-react';
+import isString from 'utils/isString';
 
 interface InputEntryProperties
-  extends React.PropsWithoutRef<JSX.IntrinsicElements['input']> {
+  extends PropsWithoutRef<JSX.IntrinsicElements['input']> {
   id: string;
-  label: string;
-  errors: object;
+  label: JSX.Element | string;
+  errorMessage: string | undefined;
```

Comment:
For consistency I vote we use `optional` 
```suggestion
  errorMessage?: string;
```